### DISCUSSION TITLE
menu for user (not administrator)

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6850] [Help]
Hi all,

I've disabled user permission on user form, because I want permission only on roles.

A added new user with role _Administrator_.

I assigned permission to _Administrator_ role on all new tables (all table are under Administration module)

If I log in with the new user credential, no menu is displayed.

What I'm missing?

Thank you,
Elena.

**Comment by edwardch** [comment]
I have also been experiencing this. 

Can't say _exactly_ when it happens, but there is definitely an issue with either - users being assigned new roles, OR, roles being assigned new/different permissions. 
_Edit: users have reported this happening when adding roles to users._

Seems that the users permission cache doesn't get invalidated, so menu's don't change until I explicitly edit a permission on the user directly, then save.
